### ISSUE TITLE
missing space

### ISSUE THREAD

**Issue by leonardpitzu** [boredazfcuk/docker-icloudpd#666]
**Describe the bug**
the container cannot start due to a missing space

**Expected behaviour**
container should start

**Actual behaviour**
container crashes

**System (please complete the following information):**
 - OS: DSM
 - Version: 7.2

**Debug log file**
`ash: missing ]`

`if [ "${webhook_insecure}"]; then` is the culprit

**Comment by leonardpitzu** [comment]
This is indeed weird...
Short q: does the docker hub image contain the fix? If I get both the latest tag and latest itself I still see the typo and thus the container fails to start and errors out with code 60 (I have startup notifications enabled)

**Comment by boredazfcuk** [comment]
> This is indeed weird...
> 
> Short q: does the docker hub image contain the fix? 

Yes. I pushed it two days ago. I've also confirmed in a running container just now to double check.

> If I get both the latest tag and latest itself I still see the typo and thus the container fails to start and errors out with code 60 (I have startup notifications enabled)

Are you sure you've pulled the latest image from Dockerhub? If you have an old "latest" in your image cache, Docker will just use that one. You either need to delete the image from the cache, or tell Docker to pull the latest version.

**Comment by leonardpitzu** [comment]
I removed the images and did a fresh pull and still the typo was in there. Only thing left is to reboot the machine...

**Comment by leonardpitzu** [comment]
tested on a clean machine - typo still there in the `:latest`

**Comment by boredazfcuk** [comment]
Nope. Latest is also tagged 1.0.982_13-10-2024:
```
boredazfcuk@Server:~> docker run --rm boredazfcuk/icloudpd:1.0.982_13-10-2024 grep 'webhook_insecure' /usr/local/bin/sync-icloud.sh
Unable to find image 'boredazfcuk/icloudpd:1.0.982_13-10-2024' locally
1.0.982_13-10-2024: Pulling from boredazfcuk/icloudpd
Digest: sha256:d167245d3cb4f65dfbf6a35faa171ee8215374f1f0fed96c58b6f4cd9424097d
Status: Downloaded newer image for boredazfcuk/icloudpd:1.0.982_13-10-2024
         if [ "${webhook_insecure}" ]; then
```
Definitely has the space in 1.0.982_13-10-2024, which is latest.

**Comment by leonardpitzu** [comment]
There was another typo on line 1724 - insecure was spelled wrong - there it fails.

**Comment by boredazfcuk** [comment]
Fixed and pushed 1.0.984_15-10-2024
